httpserver/Dockerfile

```diff
@@ -1,4 +1,3 @@
-ARG BUILDPLATFORM="linux/amd64"
```

Comment:
just out of curiosity, what if we keep this line there? Would it make a difference to the multi-arch image build?

Reply:
yes, removing this line was required, i m guessing other wise its always setting to amd64.